core/domain/exp_domain.py

```diff
@@ -47,6 +47,7 @@
 STATE_PROPERTY_CONTENT = 'content'
 STATE_PROPERTY_CONTENT_IDS_TO_AUDIO_TRANSLATIONS = (
     'content_ids_to_audio_translations')
+STATE_PROPERTY_CONTENT_TRANSLATIONS = 'content_translations'
```

Comment:
I'm not sure about the name "content translations". "Content" refers to a specific part of the state (the initial text area). I think we need another name to cover all the different parts of a state that are translatable.

Written translations perhaps?